src/components/dialogmote/referat/Referat.tsx

```diff
@@ -349,16 +372,78 @@ const Referat = ({
                 <span>{savedReferatText(lastSavedTime.toDate())}</span>
               </div>
             )}
-            <ReferatFritekster dialogmote={dialogmote} mode={mode} />
+            <div className="flex flex-col gap-8 mb-8">
+              {mode === ReferatMode.ENDRET && (
+                <ReferatTextArea
```

Comment:
Bruker `<ReferatTextArea>` for hvert fritekstfelt direkte herfra i stedet for at det er "skjult" i `<ReferatFritekster>`